### ISSUE TITLE
Email notification content updates

### ISSUE THREAD

**Issue by JerPearson** [bcgov/SIMS#3724]
Content updates to outgoing email notifications

**Acceptance Criteria**

- [x] 3.1 - Assessment Ready

> Assessment Update - StudentAid BC

> Your application has been assessed and is ready for review.  Please log in to your StudentAid BC account to view the 
assessment details.

- [x] 3.2 - Disbursements blocked by federal/eCert
> Application Update - StudentAid BC

> Your StudentAid BC funding is currently on hold.  Please log in to your StudentAid BC account for more information. 

- [x] 3.3 - Institution complete confirmation of enrolment 
> Confirmation of Enrolment Update

> Your institution has completed your confirmation of enrolment.  Please log in to your StudentAid BC account for more information.

- [x] 3.4 - Institution completes PIR
> Program Information Update - StudentAid BC

> Your institution has completed your program information request.  Please log in to your StudentAid BC account for more information.

- [x] 3.5 - Institution reports a change
>Application Update - StudentAid BC

> Your institution has reported a change to your application.  Please log in to your StudentAid BC account for more information.

- [x] 3.6 - Institution requests a change
> Action Required - StudentAid BC

> Your institution has requested a change to your application that requires action.  Please log in to your StudentAid BC account for more information.

- [x] 3.7 - Ministry Completes Application Offering Change Request 
> Offering Change Update - StudentAid BC

> The application change requested by your institution has been processed by StudentAid BC, and further action may be required.  Please log in to your StudentAid BC account for more information.

- [x] 3.8 - Ministry completes exception 
> Exception Request Update - StudentAid BC

> Your exception request has been processed by StudentAid BC, and further action may be required.  Please log in to your StudentAid BC account for more information.

- [x] 3.9 - Ministry completes requested change
> Request a Change Update

> Your requested change has been processed by StudentAid BC, and further action may be required.  Please log in to your StudentAid BC account for more information.

- [x] 3.10 - Ministry File Upload
> Account Update - StudentAid BC

> A file has been uploaded to your account on ((date)).  You can view the file on the File Uploader page of your StudentAid BC account.

- [x] 3.11 - MSFAA Cancelled
> Account Update - StudentAid BC

> Your Master Student Financial Assistance Agreement (MSFAA) has been cancelled by the National Student Loan Service Centre (NSLSC), restricting you from receiving further funding until resolved.   Please log in to your StudentAid BC account for more information.

- [x] 3.13 - Restriction added
> Account Update - StudentAid BC

> A restriction has been placed on your account.  Please log in to your StudentAid BC account for more information.

- [x] 3.14 SIN validation complete
> Account Update - StudentAid BC

> Your StudentAid BC account SIN validation is now complete.  Please log in to your StudentAid BC account for more information.  

- [x] 3.15 - Supporting user information required
> Action Required - StudentAid BC

> Your application is currently on hold because the supporting information from your ((supportUserType)) has not yet been submitted.  This information must be submitted before your application can be assessed.  Please log in to your StudentAid BC account for more information.

- [x] 3.16 - Action required disability
> Action Required - StudentAid BC
> Your application ((applicationNumber)) requires action regarding your disability status.  Please log in to your StudentAid BC account for more information.

**Comment by JerPearson** [comment]
@CarlyCotton email template content updated.  Reviewed ministry emails with SD and they are happy with basic info currently there.

**Comment by CarlyCotton** [comment]
Closing - all updated
